### PR TITLE
Add check for readme to travis

### LINKED ISSUE #337: add check for readme to travis
The line to use is:

```sh
twine check dist/*
```

### PR DESCRIPTION
**Pull request by yathomasi**
Added check for readme to travis using twine. It closes #337

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,9 +15,16 @@ install:
     - conda create -n testenv --yes pip python=$PYTHON
     - source activate testenv
     - conda install --yes --file requirements.txt
+    - if [ $PYTHON == 3.5 ];
+      then pip install twine==1.15.0;
+      else conda install --yes twine;
+      fi
```
Comment (jasmainak) [inline .travis.yml]: that's a bit weird. Couldn't we just do:

```sh
pip install --upgrade twine
```
or something like that?

**Comment by yathomasi** [comment]
@jasmainak there was dependency problem with conda install on python 3.5 so I tried to bypass that with  if else where simple pip upgrade was just fine.

File: README.rst [hunk 2]
```diff
@@ -8,9 +8,8 @@ A python implementation of elastic-net regularized generalized linear models
 
 `[Documentation (stable version)]`_ `[Documentation (development version)]`_
 
-.. raw:: html
-
-    <img src="pyglmnet-logo.png" height="400px">
+.. image:: https://raw.githubusercontent.com/glm-tools/pyglmnet/master/pyglmnet-logo.png
+   :height: 400px
```
Comment (jasmainak) [inline README.rst]: @pavanramkumar what do we do about this?

The height parameter is unfortunately not respected by github which is why we had to use the raw::html directive

Comment (jasmainak) [reply README.rst]: let's maybe use this so we don't have to use the `height` parameter:

pyglmnet

The url would be: https://user-images.githubusercontent.com/15852194/67919367-70482600-fb76-11e9-9b86-891969bd2bee.jpg

Comment (pavanramkumar) [reply README.rst]: @jasmainak seems like a reasonable solution to use a smaller image. is that was you proposed?

**Comment by jasmainak** [comment]
okay I pushed a commit. @pavanramkumar feel free to merge once CIs come green.

I'm also updating the wiki to remove the instruction about the ..raw::html line

**Comment by jasmainak** [comment]
Merged, thanks @yathomasi !

**Pull request merged**